Vic2ToHoI4/Source/HOI4World/HoI4Country.h

```diff
@@ -203,6 +203,16 @@ class Country
 
 		std::set<std::string>& editAllies() { return allies; }
 
+		[[nodiscard]] std::map<std::string, int> getGPInfluences() const { return GPInfluences; }
```

Comment:
Looks like it's not used outside the class, remove.

Reply:
Done